Library/Formula/libdecnumber.rb

```diff
@@ -0,0 +1,41 @@
+class Libdecnumber < Formula
+  desc "ANSI C implementation of General Decimal Arithmetic"
+  homepage "http://speleotrove.com/decimal/#decNumber"
+  url "http://download.icu-project.org/files/decNumber/decNumber-icu-368.zip"
+  version "3.68"
+  sha256 "14ec2cf30b58758493a7661b78b80abfb281652b61a425b85cda83173518fe25"
+  resource "cmakelists" do
```

Comment:
We could install those with `lib.install` as well. We're not likely to accept a build process not supported by upstream because if it breaks and we upstream it we'll get slightly perplexed looks. If you want to use `cmake` it should be supported by them.